### PR TITLE
Redesign of the Launch token page

### PR DESCRIPTION
**Pull request by saidam90**
- Updated the design of the "Launch token" page
- Updated validation

### CODE REVIEW

File: app/globals.css [hunk 1]
```diff
@@ -173,5 +173,5 @@ input.file-input-with-big-plus[type="file"]::file-selector-button {
 
 /* To hide the 'Choose file' label on the Create your token page as it's not customizable  */
 #image::-webkit-file-upload-button {
-  display: none;
+  visibility: hidden;
```
Comment (SiegfriedBz) [inline app/globals.css:176]: I m not sure why you changed it, I think we want to hide and not keep the space of the default file-upload-btn ?

display: none;: Completely removes the element from the layout (no space).
visibility: hidden;: Hides the element but maintains its space in the layout.

**Comment by SiegfriedBz** [comment]
sounds all good to me ! :) 
except the default file btn css for which I have a doubt

@dcts could you please double check this PR ?

File: app/globals.css (hunk 1 above)
Comment (saidam90) [reply app/globals.css:176]: Yes, but with `display:none` the "No file chosen" line is still there for some reason and isn't customisable - please see the screenshot below
Screenshot 2024-10-14 at 17 31 30

Only with `visibility: hidden` I managed to hide it to make it consistent with the design. The screenshot is below:
Screenshot 2024-10-14 at 17 32 49

Comment (dcts) [reply app/globals.css:176]: visibility hidden is fine :)

**Review by dcts — CHANGES_REQUESTED** [review]
Looks great, please just run `npm run lint` and fix any errors:

Bildschirmfoto 2024-10-14 um 15 38 59

**Pull request merged**